src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(num) {
+      this.result += num;
+
+      return this;
+    },
+    subtract(num) {
+      this.result -= num;
+
+      return this;
+    },
+    multiply(num) {
+      this.result *= num;
+
+      return this;
+    },
+    divide(num) {
+      this.result /= num;
+
+      return this;
+    },
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    operate(operation, num) {
+      this.result = operation(this.result, num);
```

Comment:
you forget to bind operation function to `this`
```suggestion
      this.result = operation.call(this, this.result, num);
```